src/node/mini_miner.cpp

```diff
@@ -264,14 +265,16 @@ void MiniMiner::BuildMockTemplate(std::optional<CFeeRate> target_feerate)
         // Pick highest ancestor feerate entry.
         auto best_iter = m_entries.begin();
         Assume(best_iter != m_entries.end());
-        const auto ancestor_package_size = (*best_iter)->second.GetSizeWithAncestors();
```

Comment:
In  [mini miner]: Linearize should also return packages fees and sizes:

What's the reason behind changing from `auto` to `uint64_t` here?

Reply:
Because it will implicitly be `int64_t` and we changed `CFeerate` size to `uint64_t` in https://github.com/willcl-ark/bitcoin/pull/15/commits/f830cea6337d7e842077b8ef01bf93a5a688ab1f, I think in this case it's better to just explicitly declare in `uint64_t`.
Than to use auto and `static_cast`?